packages/airview-compliance-ui/src/features/application-tile/application-tile-call-to-action-button.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@mui/material/Button";
```

Comment:
For consistency, import as a named import 

import { Button } from "@mui/material";